apps/coursition/src/components/transcriptionResults.tsx

```diff
@@ -0,0 +1,59 @@
+import { Tabs, TabsContent, TabsList, TabsTrigger } from '@nmit-coursition/ui/primitives'
+import { CheckIcon, CopyIcon } from 'lucide-react'
+import { useState } from 'react'
+
+interface Props {
+  raw: string
+  srt: string
+  vtt: string
+}
+
+export const TranscriptionResults = ({ raw, srt, vtt }: Props) => {
+  return (
+    <Tabs defaultValue='raw' className='w-full'>
+      <TabsList className='w-full'>
+        <TabsTrigger value='raw'>raw</TabsTrigger>
+        <TabsTrigger value='srt'>srt</TabsTrigger>
+        <TabsTrigger value='vtt'>vtt</TabsTrigger>
+      </TabsList>
+      <TabsContent value='raw'>
+        <TextCopier text={raw} title='Transcript' />
+      </TabsContent>
+      <TabsContent value='srt'>
+        <TextCopier text={srt} title='SRT subtitles' />
+      </TabsContent>
+      <TabsContent value='vtt'>
+        <TextCopier text={vtt} title='VTT subtitles' />
+      </TabsContent>
+    </Tabs>
+  )
+}
+
+const TextCopier = ({ text, title }: { text: string; title: string }) => {
```

Comment:
**suggestion:** This should be in separate pull request and the implementation should be probably different. Please remove it from here.